### PR TITLE
Upgrading to new showyourwork

### PR DESCRIPTION
**Pull request by dfm**
cc @rodluger

### CODE REVIEW

File: showyourwork.yml [hunk 1]
```diff
@@ -1,18 +1,27 @@
-# User configuration; see the docs for details!
+version: 0.3.0.dev0
 verbose: true
+rule_fallback: skip
```
Comment (rodluger) [inline showyourwork.yml:3]: I got rid of this setting; won't exist in v0.3.0

Comment (dfm) [reply showyourwork.yml:3]: One of the error messages told me to add it :D

Comment (rodluger) [reply showyourwork.yml:3]: Just get rid of it when you upgrade to the actual release. I found a much better way of dealing with missing input files -- you can probably find it if you poke around the tip of the dev branch. Snakemake raises an error for files with no "producers" but catches the exception if the file exists on disk. Showyourwork now re-raises the exception when that happens. User can override this behavior -- see config docs. I'll put a new prerelease on pypi for you tomorrow with the new syntax

**Comment by rodluger** [comment]
Looks like you figured everything out! Is it building?

**Comment by dfm** [comment]
It is: https://github.com/one-datum/paper/actions/workflows/build.yml

It was pretty straightforward to update and the `datasets` feature is excellent.

**Pull request merged**